src/components/org/OrgAdmin/CommonSection/BrandingSubColor.tsx

```diff
@@ -0,0 +1,78 @@
+import Image from 'next/image';
+import { useState } from 'react';
+
+import {
+  StColorPreview,
+  StColorWrapper,
+  StDescription,
+  StInfoButton,
+  StInfoCloseButton,
+  StInfoDescription,
+  StInfoSubDescription,
+  StInfoTitle,
+  StInfoWrapper,
+  StInput,
+  StInputBox,
+  StInputLabel,
+} from './style';
+
+interface BrandingSubColorProps {
+  subGrayColor: string;
+  onSetSubGrayColor: (color: string) => void;
+}
+
+const BrandingSubColor = ({
+  subGrayColor,
+  onSetSubGrayColor,
+}: BrandingSubColorProps) => {
+  const [isInfoVisible, setIsInfoVisible] = useState(false);
+
+  const handleInfoToggle = () => {
+    setIsInfoVisible((prev) => !prev);
+  };
+
+  return (
+    <StInputBox>
+      <StInputLabel as={'p'}>
+        <label htmlFor="sub-color">서브 컬러 (강조 그레이 컬러)</label>
+        <StInfoButton onClick={handleInfoToggle}>&nbsp;&#9432;</StInfoButton>
+      </StInputLabel>
+      <StDescription>
+        강조하고 싶은 박스의 그레이 컬러를 지정해주세요.
+      </StDescription>
+      <StColorWrapper>
+        <StInput
+          id="sub-color"
+          type="text"
+          placeholder="ex. #ffffff"
+          value={subGrayColor}
+          onChange={(e) => onSetSubGrayColor(e.target.value)}
+        />
+        <StColorPreview style={{ backgroundColor: subGrayColor }} />
+        <StInfoWrapper isVisible={isInfoVisible}>
+          <StInfoTitle>
+            <span>&#9432; 서브컬러 (강조 그레이 컬러) 예시</span>
+            <StInfoCloseButton onClick={handleInfoToggle}>
+              &#10005;
+            </StInfoCloseButton>
+          </StInfoTitle>
+          <StInfoDescription>
+            &#39;지원하기 &#39; 탭 속 파트별 소개 &#39;👍이런 분이면
```

Comment:
```suggestion
            &#39;지원하기&#39; 탭 속 파트별 소개 &#39;👍이런 분이면
```

<img width="228" alt="image" src="https://github.com/user-attachments/assets/b3bc7c3e-f2f8-4b91-b757-f93760acf50d">